src/Components/CommentsWidget.js

```diff
@@ -259,7 +259,17 @@ export class CommentForm extends Component {
                 <div className="clearfix" />
 
                 <div className="col25 col100-mobile">
-                    <strong>
+                    <strong
+                        title={`The following tags are supported:
```

Comment:
Per the request in the issue. This is the most subtle way of adding a tooltip to this widget. 

If the comment needs to be internationalized, then I request help in how to go about that.

I did not see any tooltip libraries being used in this repo, but but we can use one if you wish, or add a (i) icon. (but an icon isn't very subtle)

Reply:
Thank you! I think this kind of tooltip is sufficient, but I think it would make more sense if the tooltip would appear on the field it self as well. 

Use the `t()` function for internationalisation, like here: 

https://github.com/duaimei/datarequests-website/blob/8c54b4145f14486b6b9df238360169b55046f391/src/Components/CommentsWidget.js#L53

 and add the string in `src/i18n/en.json`